### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by fujunwei**

### CODE REVIEW

**Comment by fujunwei** [comment]
@huningxin @lincsoon PTAL.

**Comment by huningxin** [comment]
LGTM.

**Comment by lincsoon** [comment]
LGTM

**Comment by rakuco** [comment]
please don't merge yet, there's some proof-reading that needs to be done

File: README.md [hunk 1]
```diff
@@ -23,13 +23,19 @@ instead of the System WebView. Requires cordova-android 4.0 or greater.
 
 The following directions are for cordova-cli (most people).  Alternatively you can use the [Android platform scripts workflow](PlatformScriptsWorkflow.md).
 
-* Open an existing cordova project, with cordova-android 4.0.0+, and using the latest CLI (If below Cordova 5.2.0 which support default variable, please append those values like `--variable XWALK_VERSION="14+" --variable XWALK_COMMANDLINE="--disable-pull-to-refresh-effect" --variable XWALK_MODE="embedded"`).
+* Open an existing cordova project, with cordova-android 4.0.0+, and using the latest CLI. Crosswalk variables can be configurable as a soft option when installing the plugin
```
Comment (rakuco) [inline README.md]: "configured", not "configurable". What is a "soft option"?

File: README.md [hunk 2]
```diff
@@ -23,13 +23,19 @@ instead of the System WebView. Requires cordova-android 4.0 or greater.
...
 * Add this plugin
 
 ```
 $ cordova plugin add cordova-plugin-crosswalk-webview
 ```
 
+If the version of CLI is below Cordova 5.2.0, these default variables need be appended at install time
```
Comment (rakuco) [inline README.md]: If these are defaults, they shouldn't need to be specified, right? How about just saying "these variables"?

Comment (rakuco) [inline README.md]: "need to be", not "need be"

File: README.md [hunk 3]
```diff
@@ -55,21 +61,48 @@ To build System-webview apk, remove this plugin and run:
 
 ### Configure
 
-You can try out a different Crosswalk version using a `<preference>` tag within your `config.xml`. Some examples:
+You can try out a different Crosswalk version by specify certain variables at install time, and changing the value of `xwalkVersion` in your `config.xml` after installing plugin. Some examples:
```
Comment (rakuco) [inline README.md]: "by specifying"
"after installing the plugin"

File: README.md [hunk 4]
```diff
@@ -55,21 +61,48 @@ To build System-webview apk, remove this plugin and run:
...
+    --variable XWALK_VERSION="org.xwalk:xwalk_core_library:14+"
+    --variable XWALK_VERSION="xwalk_core_library:14+"
+    --variable XWALK_VERSION="14+"
+    --variable XWALK_VERSION="14"
     <preference name="xwalkVersion" value="org.xwalk:xwalk_core_library:14+" />
     <preference name="xwalkVersion" value="xwalk_core_library:14+" />
     <preference name="xwalkVersion" value="14+" />
     <preference name="xwalkVersion" value="14" />
 
+Also You can use Crosswalk beta version. Some examples:
```
Comment (rakuco) [inline README.md]: "You can also use a Crosswalk beta version"

File: README.md (hunk 2 above)
Comment (rakuco) [inline README.md]: "If your Cordova CLI version is lower than 5.2.0"

File: README.md [hunk 5]
```diff
@@ -55,21 +61,48 @@ To build System-webview apk, remove this plugin and run:
...
 ### Release Notes
 
+#### 1.3.0 (August 28, 2015)
+* Crosswalk variables can be configurable as a soft option via latest CLI, but those default variables must be appended below Cordova 5.2.0
+
+```
+$ cordova plugin add cordova-plugin-crosswalk-webview --variable XWALK_VERSION="15+"
+```
+
+* Shared mode will be used if confirguring the `XWALK_MODE` to `shared`, more information about shared mode you can get from https://crosswalk-project.org/documentation/shared_mode.html
```
Comment (rakuco) [inline README.md]: "if the XWALK_MODE option is set to shared"
"more information about Crosswalk's shared mode is available at"

File: README.md (hunk 2 above)
Comment (fujunwei) [inline README.md]: If Cordova CLI version is lower than 5.2.0, these default variables still need to be specified when installing the plugin.
If Cordova CLI version is upper than 5.2.0, they shouldn't need to be specified.

**Comment by fujunwei** [comment]
@rakuco Thanks, Done.

File: README.md [hunk 6]
```diff
@@ -55,21 +61,48 @@ To build System-webview apk, remove this plugin and run:
...
+    --variable XWALK_VERSION="org.xwalk:xwalk_core_library:14+"
+    --variable XWALK_VERSION="xwalk_core_library:14+"
+    --variable XWALK_VERSION="14+"
+    --variable XWALK_VERSION="14"
     <preference name="xwalkVersion" value="org.xwalk:xwalk_core_library:14+" />
     <preference name="xwalkVersion" value="xwalk_core_library:14+" />
     <preference name="xwalkVersion" value="14+" />
     <preference name="xwalkVersion" value="14" />
 
+You can also use Crosswalk beta version. Some examples:
```
Comment (rakuco) [inline README.md]: "a Crosswalk beta version"

File: README.md [hunk 7]
```diff
@@ -55,21 +61,48 @@ To build System-webview apk, remove this plugin and run:
...
+You can use [shared mode](https://crosswalk-project.org/documentation/shared_mode.html) that allows multiple Crosswalk applications to share one Crosswalk runtime.
+
+    <!-- These are all equivalent -->
+    --variable XWALK_MODE="shared"
+    <preference name="xwalkMode" value="shared" />
+
 ### Release Notes
 
+#### 1.3.0 (August 28, 2015)
+* Crosswalk variables can be configured as a option via latest CLI, but those default variables must be specified if CLI is lower than Cordova 5.2.0
```
Comment (rakuco) [inline README.md]: "as an option"
"if the Cordova CLI version is lower than 5.2.0"

**Comment by fujunwei** [comment]
@rakuco Thanks, Done.

**Comment by rakuco** [comment]
lgtm, thanks

**Pull request merged**